runtime/compiler/runtime/J9Profiler.cpp

```diff
@@ -589,44 +589,7 @@ void TR_ValueProfiler::modifyTrees()
                addProfilingTrees(firstChild, tt->getPrevTreeTop(), 10);
             }
          }
-
-
-      if (!getInitialCompilation())
-         visitNode(node, tt, visitCount);
-      }
-   }
-
-void
-TR_ValueProfiler::visitNode(
```

Comment:
`visitNode` only exists to handle long parm profiling, which hasn't been enabled in over 10 years and has been removed from OMR.  Hence, the entire `visitNode` function is removed.